hq/app/views/iam/iamCredReportBody.scala.html

```diff
@@ -0,0 +1,49 @@
+@import logic.DateUtils
+@import model.CredentialReportDisplay
+@import logic.ReportDisplay
+@(report: CredentialReportDisplay)
+
+<div class="container">
```

Comment:
Suggestion:

For the layout, I would take the greedy-width class off the [main container](https://github.com/guardian/security-hq/pull/17/files#diff-a704b7bf68dfc593b988548cb8d85c3cR16) of the IAM page, and place it on this DIV instead. This will mean the accordion width matches the header, but the tables inside each account can grab some extra width

Reply:
done